charts/ceph-csi-rbd/values.yaml

```diff
@@ -87,7 +87,7 @@ nodeplugin:
   plugin:
     image:
       repository: quay.io/cephcsi/cephcsi
-      tag: canary
+      tag: 2.1.0
```

Comment:
@humblec this need to be v2.1.0. that why CI is failing

Reply:
awesome, we would have made same issue as https://github.com/ceph/ceph-csi/issues/829. as we have E2E for helm we got this issue

Reply:
done.. thanks @Madhu-1